api/data_refinery_api/views.py

```diff
@@ -540,7 +541,12 @@ def perform_update(self, serializer):
             accessions = new_data['data'][key]
             if accessions == ["ALL"]:
                 experiment = get_object_or_404(Experiment, accession_code=key)
-                sample_codes = list(experiment.samples.filter(is_processed=True).values_list('accession_code', flat=True))
+
+                # QN organisms
+                organism_ids = list(ComputationalResultAnnotation.objects.filter(data__is_qn=True).values_list('data__organism_id', flat=True).order_by('data__organism_id'))
+                organisms = Organism.objects.filter(id__in=organism_ids)
```

Comment:
What about calculating this outside of the `for...loop` above? In some cases we send multiple `ALL` keys:

![image](https://user-images.githubusercontent.com/1882507/54136838-e2ef7000-43f2-11e9-9c34-9fb4424e2baf.png)